samza-core/src/main/scala/org/apache/samza/metrics/reporter/MetricsSnapshotReporter.scala

```diff
@@ -106,9 +106,20 @@ class MetricsSnapshotReporter(
     }
   }
 
-  def run {
-    debug("Begin flushing metrics.")
+  def run() {
```

Comment:
Do you think it'd be possible to add a test to test the emission logic in TestMetricsSnapshotReporter?
Somehow it was overlooked before.

Reply:
Done